tests/lib/nested.sh

```diff
@@ -328,6 +328,10 @@ configure_cloud_init_nested_core_vm_uc20(){
     umount "$tmp"
 }
 
+get_nested_core_image_path(){
```

Comment:
that image you should use is ubuntu-core-new.img

Reply:
As discussed I need the image that got created by create_nested.. helper. The -new image is a copy created late by start nested helper.